crates/sui-graphql-rpc/src/types/owner.rs

```diff
@@ -36,7 +36,7 @@ use sui_types::gas_coin::GAS;
     field(
         name = "balance",
         ty = "Option<Balance>",
-        arg(name = "type", ty = "Option<String>")
+        arg(name = "type", ty = "Option<ExactTypeFilter>")
```

Comment:
this is really swell, glad you figured out how to process input strings into a struct